pkg/jmxfetch/jmxfetch.go

```diff
@@ -211,11 +225,20 @@ func (j *JMXFetch) Start(manage bool) error {
 	if err != nil {
 		return err
 	}
+
+	// don't pollute the JSON with the log pattern.
+	out := log.Info
```

Comment:
A configurable pluggable output would be much cleaner IMO

Reply:
For `out`? For example to send the output in a file or anything?
Are you fine with planning this in another PR? This was thrown on `log.Info` before this one.